auth0/src/main/java/com/auth0/android/authentication/ParameterBuilder.java

```diff
@@ -153,8 +171,11 @@ public ParameterBuilder setAudience(@NonNull String audience) {
      *
      * @param device a device name
      * @return itself
+     *
+     * @deprecated TODO
```

Comment:
Reason is RO being legacy API being removed in the next major
That's RO https://auth0.com/docs/api/authentication#database-ad-ldap-active-

Reply:
Pushed a new commit to add this.